index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formulário</title>
+</head>
+<body>
+    <h1>Time de Valorant</h1>
+    <p>Formulario para montar um time de valorant</p>
+    <form action="mailto:alefstarkin@gmail.com">
+        <div>
+            <label>Nome:</label><input type="text" placeholder="Digite seu nome" required>
+            <p></p>
```

Comment:
Eu imagino que vc tenha utilizado o elemento `p` para gerar uma quebra de linha entre os inputs, para melhorar o visual.
Vale lembrar q não é responsabilidade do HTML lidar com o visual de uma página, isso fica a cargo do css.
Dito isso, existem elementos mais apropriados para esse mesmo fim, como o `<br/>`

```suggestion
            <br/>
```